esp/src/src-react/components/Filter.tsx

```diff
@@ -105,37 +123,6 @@ const fieldsToRequest = (fields: Fields) => {
     return retVal;
 };
 
-const TargetClusterTextField: React.FunctionComponent<TextFieldProps> = (props) => {
-
-    const [targetClusters, setTargetClusters] = React.useState<TpLogicalClusterQuery.TpLogicalCluster[]>([]);
-
-    React.useEffect(() => {
-        const topology = new Topology({ baseUrl: "" });
-        topology.fetchLogicalClusters().then(response => {
-            setTargetClusters([{ Name: "", Type: "", LanguageVersion: "", Process: "", Queue: "" }, ...response]);
-        });
-    }, []);
-
-    return <TextField {...props} >
-        {targetClusters.map(tc => <MenuItem key={tc.Name} value={tc.Name}>{tc.Name}{tc.Name !== tc.Type ? ` (${tc.Type})` : ""}</MenuItem>)}
-    </TextField>;
-};
-
-export const TargetGroupTextField: React.FunctionComponent<TextFieldProps> = (props) => {
-
-    const [targetGroups, setTargetGroups] = React.useState([]);
-
-    React.useEffect(() => {
-        TpGroupQuery({}).then(({ TpGroupQueryResponse }) => {
-            setTargetGroups(TpGroupQueryResponse.TpGroups.TpGroup);
-        });
-    }, []);
-
-    return <TextField {...props} >
-        {targetGroups.map(tc => <MenuItem key={tc.Name} value={tc.Name}>{tc.Name}{tc.Name !== tc.Kind ? ` (${tc.Kind})` : ""}</MenuItem>)}
-    </TextField>;
-};
-
```

Comment:
TargetClusterTextField and TargetGroupTextField need to be their own components (as they will be used in other dialogs).